Makefile

```diff
@@ -44,11 +53,11 @@ $(DIST)/specs/%.yaml: $(ROOT)/%.yaml $(SCHEMAS)
 	$(REDOCLY) bundle $(REDOCLY_BUNDLE_FLAGS) $< --output=$@
 
 .PHONY: lint
-lint: $(GOMPLATE_FINAL) $(SCHEMAS_FINAL)
+lint: resource-apis $(SCHEMAS_FINAL)
```

Comment:
I think the lint currently doesn't work because the files are only generated after the `resource-apis` task, so `(SCHEMAS_FINAL)` is empty